components/ArticleCard.js

```diff
@@ -0,0 +1,32 @@
+import { Component } from 'react'
+import Link from 'next/link'
+import moment from 'moment'
+
+class ArticleCard extends Component {
+  truncate (string) {
+    if (string.length > 99) {
+      return `${string.substr(0, 99)} ...`
+    } else {
+      return string
+    }
+  }
+
+  render () {
+    const { article } = this.props
+
+    return <li className="c-articles__list">
+      <img className="c-articles__list--hero" src={`http://localhost:3001${article.previewImage}`} />
```

Comment:
`http://localhost:3001` put this into an environment variable so that it can be changed depending on whether its in development or production